tools/plugins/com.liferay.ide.upgrade.plan.ui/META-INF/MANIFEST.MF

```diff
@@ -27,6 +27,7 @@ Require-Bundle: com.liferay.ide.project.core,
  org.eclipse.ui.navigator,
  org.eclipse.ui.navigator.resources,
  org.eclipse.ui.workbench,
- org.eclipse.ui.workbench.texteditor
+ org.eclipse.ui.workbench.texteditor,
+ org.eclipse.wst.web
```

Comment:
Hey @jtydhr88  lets not pull in the dependency on `wst.web` just in order to do this.  Because we may want to make this work without `wst.web` in the future.

Reply:
So simply copy the code to our plugin that makes this week.